app/styles/nypr-ui/_stream-banner.scss

```diff
@@ -73,16 +73,12 @@
     @include classic-filter($filter--light);
   }
 
-  @include mq($small-only) {
-    display: none;
-  }
-
```

Comment:
took this out because display at breakpoints should be managed on the parent template